index.bs

```diff
@@ -5606,9 +5613,84 @@ A <dfn>geolocation override</dfn> is a [=struct=] with:
 
 A [=remote end=] has a <dfn>geolocation overrides map</dfn> which is a weak map between [=user contexts=] and [=geolocation override=].
 
-
 ### Commands ### {#module-emulation-commands}
 
+#### The emulation.setForcedColorsModeThemeOverride Command ####  {#command-emulation-setForcedColorsModeThemeOverride}
+
+The <dfn export for=commands>emulation.setForcedColorsModeThemeOverride</dfn> command modifies
+[=forced colors mode=] theming characteristics on the given top-level traversables or user contexts.
+
+<dl>
+   <dt>Command Type</dt>
+   <dd>
+    <pre class="cddl remote-cddl">
+      emulation.SetForcedColorsModeThemeOverride = (
+        method: "emulation.setForcedColorsModeThemeOverride",
+        params: emulation.SetForcedColorsModeThemeOverrideParameters
+      )
+
+      emulation.SetForcedColorsModeThemeOverrideParameters = {
+        theme: [+emulation.ForcedColorsModeAutomationTheme],
+        ? contexts: [+browsingContext.BrowsingContext],
+        ? userContexts: [+browser.UserContext],
+      }
+
+      emulation.ForcedColorsModeAutomationTheme = "none" / "light" / "dark"
+    </pre>
+   </dd>
+   <dt>Result Type</dt>
+   <dd>
+    <code>
+     EmptyResult
+    </code>
+   </dd>
+</dl>
+
+<div algorithm="remote end steps for emulation.setForcedColorsModeThemeOverride">
+
+The [=remote end steps=] with |command parameters| are:
+
+1. If |command parameters| doesn't [=map/contain=] "<code>theme</code>",
+   return [=error=] with [=error code=] [=invalid argument=].
+
+1. If |command parameters| [=map/contains=] "<code>theme</code>",
+   and "<code>theme</code>" is not "<code>none</code>", "<code>light</code>", or "<code>dark</code>",
```

Comment:
these two steps are already covered by the CDDL validation so they can be removed

Reply:
Removed the first two steps in the latest commit. I had a clarification on the second point below, so I'll follow up on that one once I have a better understanding on what you're looking for on that one.